### DISCUSSION TITLE
I am really struggling to create a whole system using customtkinter and packing it into an exe

### DISCUSSION THREAD

**Discussion by SCFrostino** [TomSchimansky/CustomTkinter#1385] [Q&A]
I have a customtkinter system at the moment where I have multiple different python files that are linked together: for example I have a home page that says login and signup when I press one of the buttons I have a piece of code: os.system('python signup.py') I have made a whole system like this and it seems to work when I run it in my IDE (vscode) but only when I open the software as a folder, if I open the file individually vscode says it cannot find the file in the directory. if I change the directory i then get a message saying cant find __main__ module. I just want to pack all of my files into an exe because my computer science hand in is in a few days. Please someone help

**Comment by AlbertSolomon** [comment]
It's possible that the problem you're experiencing is related to the way your project's file structure is organized when it is being packaged as a software. It would be helpful to understand which software you used to bundle your project so we can better identify the root cause of the issue.

**Comment by Anirudh-12** [comment]
If you want to package the exe with custom Tkinter you must add folder of CustomTkinter  package because it contains some assets that are needed for running and also if you are using Pyinstaller you must use one folder option instead of one file option. hope this helps.
